CRM/Event/Form/Participant.php

```diff
@@ -415,15 +415,15 @@ public function setDefaultValues(): array {
 
       CRM_Event_BAO_Participant::getValues($params, $defaults, $ids);
```

Comment:
Oh it's because this is where $defaults is set, and it's indexed on id.

Reply:
thanks @demeritcowboy  - fixed